sros2/test/sros2/commands/security/verbs/test_create_key.py

```diff
@@ -80,12 +85,12 @@ def check_permissions_ca_cert_pem(path, signatory):
     assert verify_signature(cert, signatory)
 
 
-def check_req_pem(path, signatory):
+def check_req_pem(path, _):
     csr = load_csr(path)
     check_common_name(csr.subject, u'/test_node')
 
 
-def check_key_pem(path, signatory):
+def check_key_pem(path, _):
```

Comment:
Heh, this is what spawned the slight reorg in #132

Reply:
Nice :+1:, as this is handled in #132 I restricted this PR to adding the permission file validation to reduce conflicts with your PR